src/_includes/header.html

```diff
@@ -55,6 +55,7 @@
               <li><a href="https://forum.developer.concur.com/">Forum</a></li>
               <li><a href="{{ site.baseurl }}/tools-support/release-notes/index.html">Release Notes</a></li>
               <li><a href="{{ site.baseurl }}/announcements/index.html">Announcements</a></li>
+              <li><a href="{{ site.baseurl }}/announcements/videos.html">Videos</a></li>
```

Comment:
should link to tools-support instead of announcements

Reply:
Fixed link